web-api/src/app.js

```diff
@@ -263,6 +266,12 @@ const {
 const {
   getPractitionerByBarNumberLambda,
 } = require('./practitioners/getPractitionerByBarNumberLambda');
+const {
+  getPractitionerDocumentDownloadUrlLambda,
+} = require('./practitioners/getPractitionerDocumentDownloadUrlLambda');
+const {
+  getPractitionerDocumentsLambda,
+} = require('./practitioners/getPractitionerDocumentsLambda');
```

Comment:
Any idea why we're sorting single and multi-line imports separately?

Reply:
I'm not sure.  All we did was fork that existing sort plugin and fix a small syntax error.